app/controllers/monsters_controller.rb

```diff
@@ -0,0 +1,7 @@
+class MonstersController < ApplicationController
+
+  def index
+    @monsters = Monster.all
+  end
```

Comment:
Is there an ERB template using this controller? If not, it can be removed

Reply:
It leads to an index view that has a <div id="app"> to load our react.